src/QsFmt/Formatter/Rules.fs

```diff
@@ -145,3 +145,163 @@ let unitUpdate =
 
             base.Type((), updated)
     }
+
+let arraySyntaxUpdate =
+
+    let getBuiltInDefault builtIn =
+        match builtIn.Text with
+        | "Unit" -> { Prefix = []; Text = "()" } |> Literal |> Some
+        | "Int" -> { Prefix = []; Text = "0" } |> Literal |> Some
+        | "BigInt" -> { Prefix = []; Text = "0L" } |> Literal |> Some
+        | "Double" -> { Prefix = []; Text = "0.0" } |> Literal |> Some
+        | "Bool" -> { Prefix = []; Text = "false" } |> Literal |> Some
+        | "String" -> { Prefix = []; Text = "\"\"" } |> Literal |> Some
+        | "Result" -> { Prefix = []; Text = "Zero" } |> Literal |> Some
+        | "Pauli" -> { Prefix = []; Text = "PauliI" } |> Literal |> Some
+        | "Range" -> { Prefix = []; Text = "1..0" } |> Literal |> Some // ToDo: Double-check that this literal is correct
```

Comment:
I think you did get confirmation that this literal is correct, so this comment can probably be removed, right?

Reply:
I was going to see what people had to say on this review :). I'll remove the ToDo before merging, but we can keep this thread open until then as a reminder.

Reply:
Wow, I literally missed this, even after I said I wouldn't. That's my mistake, I'll remove it in my current working branch now to make sure it gets in the next one. Sorry!